docs/running-node-ansible.md

```diff
@@ -0,0 +1,204 @@
+# Running a DCLedger Node using Ansible
+
+This document describes in how to:
+
+* configure different types of DCLedger nodes: genesis, validator and observer
+* create a node administrator (or admin) account as a necessary part of validator and genesis node configuration
+
+## Requirements (local machine)
+
+### 1. Clone [DCL][5] project
+```
+git clone https://github.com/zigbee-alliance/distributed-compliance-ledger.git
+```
+
+### 2. Install `Python` and `Pip`
+```bash
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends python3
+sudo apt install python3-pip
+sudo apt install python3-testresources
+```
+
+### 3. Install [Ansible][4] and its dependencies
+    
+Run the following commands from the [DCL][5] project home
+```bash
+sudo pip install -r deployment/requirements.txt
+ansible-galaxy install -r deployment/galaxy-requirements.yml
+```
+
+## Requirements (target machine)
+
+### Hardware requirements
+
+Minimal:
+
+* 1GB RAM
+* 25GB of disk space
+* 1.4 GHz CPU
+
+Recommended (for highload applications):
+
+* 2GB RAM
+* 100GB SSD
+* x64 2.0 GHz 2v CPU
+
+### Operating System
+
+Current delivery is compiled and tested under `Ubuntu 20.04 LTS` so we recommend using this distribution for now.
+In future, it will be possible to compile the application for a wide range of operating systems (thanks to Go language).
+
+### Install `Python`
+Python3 needs to be installed on target machine to run ansible playbooks
+```bash
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends python3
+```
+
+## Configure Nodes
+
+### 1. Set network chain id in [`deployment/ansible/inventory/hosts.yml`]
+```
+all:
+  vars:
+    chain_id: <chain-id>
+  ...
+```
+Every network must have a unique chain ID (e.g. `test-net`, `main-net` etc.)
+
+### 2. Put `genesis.json` file under specific directory (if you are not running a genesis node)
+Get or download `genesis.json` file of a network your node will be joining and put it under the following path:
+```
+deployment/persistent_chains/<chain-id>/genesis.json
```

Comment:
```suggestion
```bash
wget -O deployment/persistent_chains/<chain-id>/genesis.json https://raw.githubusercontent.com/zigbee-alliance/distributed-compliance-ledger/master/deployment/persistent_chains/<chain-id>/genesis.json
```

Reply:
Genesis.json may not always exist in GitHub. These are generic instructions for NodeAdmins.

Reply:
In order to avoid creation of multiple versions of instructions, it makes sense to start with a generic item/command, and then provide an example how it will look for the Test Net (and later on for the Main Net).